README.md

```diff
@@ -16,6 +16,68 @@ This action provides the following functionality for GitHub Actions users:
 
 See [action.yml](action.yml)
 
+<!-- start usage -->
+```yaml
+- uses: actions/setup-node@v3
+  with:
+    # Version Spec of the version to use in SemVer notation.
+    # It also emits such aliases as lts, latest, nightly and canary builds
+    # Examples: 12.x, 10.15.1, >=10.15.0, lts/Hydrogen, 16-nightly, latest, node
+    node-version: ''
+
+    # File containing the version Spec of the version to use.  Examples: .nvmrc, .node-version, .tool-versions.
+    # If node-version and node-version-file are both provided the action will use version from node-version. 
+    node-version-file: ''
+
+    # Set this option if you want the action to check for the latest available version 
+    # that satisfies the version spec.
+    # It will only get affect for lts Nodejs versions (12.x, >=10.15.0, lts/Hydrogen). 
+    # Default: false
+    check-latest: false
+
+    # Target architecture for Node to use. Examples: x86, x64. Will use system architecture by default.
+    # Default: ''. The action use system architecture by default 
+    architecture: ''
+
+    # Used to pull node distributions from node-versions. Since there's a default, this is typically not supplied by the user. 
```

Comment:
```suggestion
    # Used to pull node distributions from https://github.com/actions/node-versions. Since there's a default, this is typically not supplied by the user. 
```